third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -1788,13 +1803,13 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
                         Some(entries) => {
                             if entries.len() != 1 {
                                 self.error(
-                                loc,
-                                &format!(
-                                    "Expect a unique spec function from lifted lambda: {}, found {}",
-                                    remapped_sym.display(self.symbol_pool()),
-                                    entries.len()
-                                ),
-                            );
+                                    loc,
```

Comment:
Did you insert tab characters or something?  Odd display in the diff.

Reply:
Nope, I don't see any tabs,  this is auto formatted.